Intersect.Editor/Localization/Strings.cs

```diff
@@ -1899,42 +1905,56 @@ public partial struct EventCommandList
 
             public static LocalizedString dupguildvariable = @"Guild Variable: {00}'s Value";
 
+            public static LocalizedString DupUserVariable = @"Account Variable: {00}'s Value";
```

Comment:
Ok this needs to be done too many places in this file.

Add a new struct called `GameObjects` and define `UserVariable = @"User Variable"` and `UserVariables = @"User Variables"`.

Then in every string that contains `User Variable` or `User Variables` turn them into parameterized generic strings like `AddX = @"Add {00}"` instead of `AddUserVariable = @"Add User Variable"` and then when the string is used do `AddX.ToString(GameObjects.UserVariable)` -- if the replaced string is plural, use `GameObjects.UserVariables` instead.

We have _way_ too much redundancy in our strings.

If the string is just `User Variable:` or `User Variables:` just use the regular string that doesn't have a `:` in it, we honestly use the trailing `:` wrong anyway.